src/main/java/org/broadinstitute/hellbender/tools/walkers/bqsr/BaseRecalibrator.java

```diff
@@ -98,7 +99,11 @@
      * use -XL my.interval.list to skip over processing those sites. Please note however that the statistics reported by the tool will not accurately
      * reflected those sites skipped by the -XL argument.
      */
-    @Argument(fullName = "knownSites", shortName = "knownSites", doc = "One or more databases of known polymorphic sites used to exclude regions around known polymorphisms from analysis.", optional = false)
+    @Argument(
+            fullName = "known-sites",
```

Comment:
Note that I haven't actually fixed the documentation in BaseRecalibrator I just updated a single argument name to kebab case, because it occurred in the documentation of AnalyzeCovariates